src/GraphControl/Control/Grapher.cpp

```diff
@@ -199,6 +199,16 @@ namespace GraphControl
     {
         if (auto graph = GetGraph(equation))
         {
+            for (auto graphVar : graph->GetVariables())
```

Comment:
You should modify SetGraphArgs(); to take a graph as a variable and then call that function here instead. Then update other calls to pass in m_graph so that we don't duplicate code.